javascript/futurism_channel.js

```diff
@@ -16,6 +16,10 @@ const debounceEvents = (callback, delay = 20) => {
   }
 }
 
+const targetResolver = (event) => {
```

Comment:
I have to check if this is still needed or a relic of my try & fail & try again approach